.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 ## Description
 
-Only fill in the fields below if relevant.
+Add HoleOS and its ASCII to Neofetch
```

Comment:
Please don't change the pull request template...

Reply:
You can check out this guide on how to undo a commit:
https://stackoverflow.com/questions/927358/how-do-i-undo-the-most-recent-local-commits-in-git